### ISSUE TITLE
Incorrect Execution Link in DomJudge API Documentation

### ISSUE THREAD

**Issue by TrainedPro** [DOMjudge/domjudge#2792]
### Description of the problem
The DomJudge API documentation contains an incorrect execution link, which results in a failed request when using the "Try it out" feature.

### Your environment
- **DOMjudge version**: Latest
- **Operating system**: Ubuntu 24.04
- **Webserver**: Apache
- **Configuration**: Default configuration (based on the docker-compose file)

### Steps to reproduce
1. Go to the DomJudge API documentation.
2. Click on "Try it out" for any endpoint and provide the necessary options.
3. Press the "Execute" button.

### Expected behaviour
The API request should successfully execute with the correct URL, e.g., `http://localhost/api/v4/contests/...`.

### Actual behaviour
The API request fails with the following error message:
```
Code: Undocumented
Details: Failed to fetch.

Possible Reasons:
- CORS
- Network Failure
- URL scheme must be "http" or "https" for CORS request.
```
The error occurs because the execution link contains an incorrect double `/api/` in the URL (`http://localhost//api/api/v4/contests/...`).

### Any other information that you want to share?
- This issue applies to **all** API endpoints in the documentation.
- Since the documentation is running on DomJudge itself, using `localhost` seems appropriate. However, if this needs to be deployed to different environments, it might be better to replace `localhost` with a placeholder like `{baseUrl}` for flexibility.
- The issue seems to stem from an extra `/api/` segment in the execution URL. It may be useful to adjust the generated links in the documentation to prevent this duplication.

**Comment by nickygerritsen** [comment]
I believe this is fixed in https://github.com/DOMjudge/domjudge/commit/7858e80f6eef33a749a845edd054d812b11507d9 already. Would be nice if you could test on main.

**Comment by meisterT** [comment]
@nickygerritsen note that this seems to be incorrect for demoweb as well: 
![image](https://github.com/user-attachments/assets/141e8a6e-7d15-4151-be9c-f8636edc0873)

**Comment by nickygerritsen** [comment]
Hmm that is different behavior though. But indeed that seems to be broken. Will investigate

**Comment by vmcj** [comment]
I suspect @meisterT his issue is fixed with: https://github.com/DOMjudge/domjudge/pull/2801 (we'll see tomorrow).

@TrainedPro are you running DOMjudge 8.3 or are you running main? Latest could mean both so can you specify the version so we know what to test against.

**Comment by TrainedPro** [comment]
> I suspect @meisterT his issue is fixed with: #2801 (we'll see tomorrow).
> 
> @TrainedPro are you running DOMjudge 8.3 or are you running main? Latest could mean both so can you specify the version so we know what to test against.

It is DomJudge 8.3.1, let me know if you need anymore details, happy to provide them.

**Comment by vmcj** [comment]
> > I suspect @meisterT his issue is fixed with: #2801 (we'll see tomorrow).
> > @TrainedPro are you running DOMjudge 8.3 or are you running main? Latest could mean both so can you specify the version so we know what to test against.
> 
> It is DomJudge 8.3.1, let me know if you need anymore details, happy to provide them.

And you use our docker-compose file?

**Comment by TrainedPro** [comment]
> > > I suspect @meisterT his issue is fixed with: #2801 (we'll see tomorrow).
> > > @TrainedPro are you running DOMjudge 8.3 or are you running main? Latest could mean both so can you specify the version so we know what to test against.
> > 
> > 
> > It is DomJudge 8.3.1, let me know if you need anymore details, happy to provide them.
> 
> And you use our docker-compose file? Together with our containers?

Yes, is this an issue that seems to be just for me? I have made changes to the file so it is possible that it is an error on my part.

**Comment by vmcj** [comment]
Fixed with: https://github.com/DOMjudge/domjudge/pull/2807 I intent to push a new tag (and new containers) somewhere next week after our next hackathon
